kolibri/plugins/device_management/assets/src/views/UserPermissionsPage/index.vue

```diff
@@ -1,35 +1,56 @@
 <template>
 
   <ImmersiveFullScreen v-bind="{ backPageLink, backPageText }">
+    <!-- TODO should I try and use the baked in auth page? Does this have a URL-->
     <AuthMessage v-if="!isSuperuser" authorizedRole="superuser" />
 
     <h1 v-else-if="user === null">{{ $tr('userDoesNotExist') }}</h1>
 
     <template v-else>
-      <div class="section">
+      <div class="section user-info">
         <h1>
           {{ user.full_name }}
           <span v-if="isCurrentUser">
             ({{ $tr('you') }})
           </span>
         </h1>
-        <h3>{{ user.username }}</h3>
+
+        <table>
+          <tr>
+            <th scope="row">{{ $tr('usernameLabel') }}</th>
+            <td>{{ user.username }}</td>
+          </tr>
+
+          <tr>
+            <th scope="row">{{ $tr('userTypeLabel') }}</th>
+            <td>
+              <UserTypeDisplay :userType="UserType(user)" />
+            </td>
+          </tr>
+
+          <tr>
+            <th scope="row">{{ $tr('facilityLabel') }}</th>
+            <td>{{ facilityName }}</td>
+          </tr>
+        </table>
+
       </div>
 
-      <div class="section">
+      <div class="section superuser">
         <KCheckbox
+          class="super-admin-checkbox"
           :disabled="superuserDisabled"
-          :label="$tr('makeSuperuser')"
+          :label="$tr('makeSuperAdmin')"
           :checked="superuserChecked"
           @change="superuserChecked=$event"
         />
-        <p>
-          <PermissionsIcon permissionType="SUPERUSER" class="permissions-icon" />
-          {{ $tr('makeSuperuserDetails') }}
-        </p>
-      </div>
+        <PermissionsIcon permissionType="SUPERUSER" class="permissions-icon" />
 
-      <hr>
+        <ul :class="['checkbox-description', {disabled: superuserDisabled}]">
```

Comment:
for unconditionally applied classes, simply add them to a `class` attribute to be clear that they're static:

```html
<ul class="checkbox-description" :class="{disabled: superuserDisabled}">
```